.azure-pipelines/azure-pipelines-win.yml

```diff
@@ -64,22 +64,22 @@ jobs:
 
     - task: CondaEnvironment@1
       inputs:
-        packageSpecs: 'python=3.6 conda-build conda conda-forge::conda-forge-ci-setup=2' # Optional
+        packageSpecs: 'python=3.6 conda-build=3.18.11 conda conda-forge::conda-forge-ci-setup=2' # Optional
```

Comment:
we'll see if this works...

Reply:
A good sign:
```
Detected hard-coded path in text file share/jupyter/kernels/python3/kernel.json
```

Reply:
Mind applying this to #56 so we can get the release out?